test/org/httpkit/server_test.clj

```diff
@@ -110,7 +110,10 @@
                               :body "Hello World"}))
   (GET "/iseq" [] (fn [req] {:status 200
                             :headers {"Content-Type" "text/plain"}
```

Comment:
could we fix the indentation on this `:headers` key? you've fixed the indentation for `:body` and now I can't ignore the `:headers` :stuck_out_tongue: